config/prod.exs

```diff
@@ -10,14 +10,30 @@ use Mix.Config
 # which you should run after static files are built and
 # before starting your production server.
 config :golden_path, GoldenPathWeb.Endpoint,
-  http: [port: {:system, "PORT"}],
-  # This is critical for ensuring web-sockets properly authorize.
-  url: [host: "localhost", port: {:system, "PORT"}],
   cache_static_manifest: "priv/static/cache_manifest.json",
+  code_reloader: false,
```

Comment:
It's worth noting for the purposes of a release, this all gets overwritten in rel/config/config.exs. That being said, I think its really valuable to be able to mirror a pure dev environment (i.e. with Mix only) with the release setup.